LiveSplit.AutoSplitters.xml

```diff
@@ -10220,4 +10220,15 @@
         <Description>Auto Start/Split/Reset and Load Remover (By Voxelse)</Description>
         <Website>https://github.com/Voxelse/LiveSplit.Carto</Website>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Ugh!</Game>
+        </Games>
+        <URLs>
+            <URL>https://gitlab.com/0xC0DECAFE/livesplit-auto-splitters/-/raw/master/Ugh!.asl</URL>
```

Comment:
This points to an ASL script. Please change your type to Script or point it to a component (dll).